backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -16,6 +19,27 @@
 
 
 class BibtexWriter(Writer):
+    def _encode(self, text):
+        import latexcodec  # NOQA
+
+        if "$" not in text:
+            return codecs.encode(text, "ulatex+{}".format(self.encoding))
+        else:
+            math_expression_regex = re.compile(r"(?<!\\)\$.*?\$")
+            found_math_expressions = re.findall(math_expression_regex, text)
```

Comment:
you can also do
```suggestion
            found_math_expressions = math_expression_regex.findall(text)
```
and maybe `found_math_expressions` is a bit long, what about `math_expressions`?